ui/admin/app/components/form/role/manage-org-projects/index.hbs

```diff
@@ -0,0 +1,106 @@
+{{!
+  Copyright (c) HashiCorp, Inc.
+  SPDX-License-Identifier: BUSL-1.1
+}}
+
+{{#if @model.totalItemsCount}}
+  <Rose::Form
+    class='full-width role-manage-custom-scopes-form'
+    @onSubmit={{fn @submit this.allGrantScopes}}
+    @cancel={{@cancel}}
+    @disabled={{@model.role.isSaving}}
+    as |form|
+  >
+    <Hds::SegmentedGroup as |S|>
+      <S.TextInput
+        @value={{@search}}
+        @type='search'
+        placeholder={{t 'actions.search'}}
+        aria-label={{t 'actions.search'}}
+        {{on 'input' @handleSearchInput}}
+      />
+    </Hds::SegmentedGroup>
+    {{#if @model.subScopes}}
+      {{#if this.selectedItems.length}}
+        <div class='table-selected-items-text'>
+          <Hds::Text::Body @tag='p' @color='faint'>
+            {{t
+              'resources.role.scope.titles.selected'
+              items=this.selectedItems.length
+              total=@model.totalItemsCount
+            }}
+          </Hds::Text::Body>
+        </div>
+      {{/if}}
+      <Hds::Table
+        @columns={{array
+          (hash label=(t 'resources.project.title'))
+          (hash label=(t 'form.id.label'))
+        }}
+        @model={{@model.subScopes}}
+        @isSelectable={{true}}
+        @onSelectionChange={{this.selectionChange}}
+        @valign='middle'
+      >
+        <:body as |B|>
+          <B.Tr
+            @selectionKey={{B.data.id}}
+            @isSelected={{includes B.data.id this.selectedItems}}
+            @selectionAriaLabelSuffix='row {{B.data.id}}'
+            data-test-org-scopes-table-row={{B.data.id}}
+          >
+            <B.Td>
+              <Hds::Text::Body @tag='p'>
+                {{B.data.displayName}}
+              </Hds::Text::Body>
+            </B.Td>
+            <B.Td>
+              <Hds::Copy::Snippet
+                @textToCopy={{B.data.id}}
+                @color='secondary'
+              />
+            </B.Td>
+          </B.Tr>
+        </:body>
+      </Hds::Table>
+      <Rose::Pagination
+        @model={{array @model.role.id @model.orgScope.id}}
+        @totalItems={{@model.totalItems}}
+        @currentPage={{@page}}
+        @currentPageSize={{@pageSize}}
+      />
+
+      <form.actions
+        @submitText={{t 'actions.save'}}
+        @cancelText={{t 'actions.cancel'}}
+      />
+    {{else}}
+      <Hds::ApplicationState data-test-no-scope-results as |A|>
```

Comment:
Might want to update this. We use the same `data-test-*` attribute for our [actual scopes](https://github.com/hashicorp/boundary-ui/blob/main/ui/admin/app/templates/scopes/scope/scopes/index.hbs#L101). 
```suggestion
      <Hds::ApplicationState data-test-no-grant-scope-results as |A|>
```

Reply:
great catch!